packages/webapp/app/app-boot.ts

```diff
@@ -0,0 +1,41 @@
+import type { Boot } from '@dailydotdev/shared/src/lib/boot';
+import type { DehydratedState } from '@tanstack/react-query';
+import {
+  appBootDataQuery,
+  BootApp,
+  getBootData,
+} from '@dailydotdev/shared/src/lib/boot';
+import { dehydrate } from '@tanstack/react-query';
+import { getQueryClient } from '@dailydotdev/shared/src/graphql/queryClient';
+
+interface GetAppBootDataProps {
+  cookies: string;
+}
+
+interface GetAppBootDataReturn {
+  state: DehydratedState;
+  queryClient: ReturnType<typeof getQueryClient>;
+  boot: Boot;
+}
+
+export const getAppBootData = async ({
+  cookies,
+}: GetAppBootDataProps): Promise<GetAppBootDataReturn> => {
+  const queryClient = getQueryClient();
+  const boot = await getBootData(BootApp.Webapp, undefined, { cookies });
+  await queryClient.prefetchQuery({
+    ...appBootDataQuery,
+    queryFn: async () => Promise.resolve(boot),
```

Comment:
can't we just call `getBootData`?

Reply:
you can use `fetchQuery` which prefetches and returns the data in one go.

Reply:
Nice, switching to this